src/JCache-HZInstance.md

```diff
@@ -1,15 +1,14 @@
 ## JCache - Hazelcast Instance Integration
 
-You can retrieve `javax.cache.Cache` instances directly through `HazelcastInstance::getCache(String name)` method.
-The parameter `name` in `HazelcastInstance::getCache(String name)` is the full cache name except the Hazelcast prefix, i.e., `/hz/`. 
+You can retrieve `javax.cache.Cache` instances using the interface `ICacheManager` of `HazelcastInstance`. This interface has the method `getCache(String name)` where `name` is the prefixed cache name. The prefixes in the cache name are URI and classloader prefixes, which are optional. Please note that the method `getCache(String name)` does not use a cache name with Hazelcast prefix (`/hz/`). Please see the [`ICacheManager` Javadoc](http://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/core/ICacheManager.html) for details.
 
-If you create a cache through a `CacheManager` which has its own specified URI scope (and/or specified classloader), 
-it must be prepended to the pure cache name as a prefix while retrieving the cache through `HazelcastInstance::getCache(String name)`. 
+If you create a cache through a `ICacheManager` which has its own specified URI scope (and/or specified classloader), 
+it must be prepended to the pure cache name as a prefix while retrieving the cache through `getCache(String name)`. 
 Prefix generation for full cache name (except the Hazelcast prefix, which is `/hz/`) is exposed through 
```

Comment:
Is the information about the `/hz/` prefix somehow important to users? Or is it rather an implementation detail and users will never meet this prefix?

Reply:
No, will remove it.